### PR TITLE
[screen-19 v6, issue 122] Build search screen to retrieve new food item from a given database

### PR DESCRIPTION
**Pull request by alexxispn**
## Description

This pull request adds the food item search screen to the app. This feature allows users to search for food items. The search engine displays the data that matches the characters entered while searching. Once a result is found, the user can click on it to automatically fill in the input on the food items screen. Additionally, the back arrow leads to the food item list.

## Changes

- Add routes for EntityFoodSearch.
- Add i18n text for EntityFoodSearch and linting.
- Add hook useAppInput and useEntityFoodSearch.
- Add atom component EntityFoodInput and EntityFoodNamesList.
- Update CrossIcon to receive prop Color.
- Add font mega-input.
- Update AppWrapper to include property bgColor and to increase width space on the title.
- Add page EntityFoodSearch.

## Checks

 - [X] Food items are searchable against the mocked database.
 - [X] The search engine displays matching data.
 - [X] Clicking on a result automatically fills in the input on the food items screen.
 - [X] The back arrow leads to the food item list.

## Screenshots

- EntityFoodSearch without fill

Screenshot from 2023-02-27 13-47-05

- EntityFoodSearch filled:

Screenshot from 2023-02-27 13-47-24

## Additional information

- Issue #124

### CODE REVIEW

File: src/styles/index.scss [hunk 1]
```diff
@@ -168,6 +168,14 @@
     line-height: 20px;
   }
 
+  .font-mega-input {
```
Comment (samueldominguez) [inline src/styles/index.scss:171]: can you do this with Tailwind? Also whenever you can, avoid using `px`. Tailwind will by default use `rem` units which are going to make the site look more consistent across different screens.

**Review by samueldominguez — APPROVED** [review]
LGTM

**Comment by alexxispn** [comment]
Hi @samueldominguez , thank you for the review. I just wanted to let you know that I pushed this branch targeting the pull request to the 'feature/screen-17-18-v6-build-predefined-list-of-food-items-assigned-to-specific-recipient' branch

**Comment by samueldominguez** [comment]
@alexxispn OK, are you still working on that branch? I'm guessing you plan to create a PR for that branch later, correct?

**Comment by alexxispn** [comment]
Hi @samueldominguez,  I realized that it was my mistake to target the pull request to the 'feature/screen-17-18-v6-build-predefined-list-of-food-items-assigned-to-specific-recipient' branch, as it has already been merged with main. I apologize for any inconvenience this may have caused.

To address this issue, maybe I can create a new pull request from the 'feature/screen-17-18-v6-build-predefined-list-of-food-items-assigned-to-specific-recipient' branch, with the target branch set to main. This way, the changes can be properly reviewed and merged into the main branch?

**Pull request merged**